app/models/queue_classic_admin/queue_classic_scheduled_job.rb

```diff
@@ -0,0 +1,6 @@
+module QueueClassicAdmin
+  class QueueClassicScheduledJob < ActiveRecord::Base
```

Comment:
Not sure to understand what this model is used for...as it's going to do the exact same thing as the `QueueClassicJob` model, no?